config/standard/contact-summary.templated.js

```diff
@@ -1,16 +1,30 @@
+var now = extras.now;
+var MS_IN_DAY = extras.MS_IN_DAY;
+var DAYS_IN_PNC = extras.DAYS_IN_PNC;
```

Comment:
Would destructuring make this cleaner or is being explicit better?

Reply:
Same deal :(